test/JDBC/input/functions/sys-suser_sname.sql

```diff
@@ -11,3 +11,6 @@ DECLARE @id SYS.VARBINARY(85)
 SET @id = CAST(CAST((SELECT oid FROM pg_catalog.pg_roles WHERE rolname = 'jdbc_user') AS INT) AS SYS.VARBINARY(85))
 SELECT SYS.SUSER_SNAME(@id)
 GO
+
+SELECT SYS.SUSER_SNAME(NULL)
```

Comment:
Since we have written the tests for suser_sname() in prepare/verify/cleanup format, can we have these tests in that manner and remove this file? 
The same applies for suser_sid tests.

Reply:
Done!